FBReader/src/main/java/org/geometerplus/android/fbreader/network/bookshare/BookDetailActivity.java

```diff
@@ -29,6 +37,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         requestWindowFeature(Window.FEATURE_NO_TITLE);
     }
+
+    private void determineAvailableReadingLists() {
```

Comment:
-rename method to be setHasAvailableReadingLists
- the forLoop can be extracted out into hasAvailableReadingList